### ISSUE TITLE
Create job to send mass SMS notifications 👥 

### ISSUE THREAD

**Issue by tomas-salgado** [colorstackorg/oyster#472]
### Description

We want to create a job that takes all the existing phone numbers in the database and sends a text notification utilizing [Twilio](https://www.twilio.com/en-us).

### Acceptance Criteria

- The job should take in an array of phone numbers and a message
- The job should send an SMS message to all phone numbers with the given message

### Additional Context
We will want to add this code within a new directory named `twilio` within the path `packages/core/src/modules`.

This will be the first integration of Twilio with Oyster(!) so this will require some research-here are the Twilio [docs](https://www.twilio.com/docs#twilio-essentials). Feel free to check out the [Slack use-cases](https://github.com/colorstackorg/oyster/tree/main/packages/core/src/modules/slack/use-cases) to see examples of Slack jobs.

**Comment by Habeebah157** [comment]
I would like to do this one.

**Comment by Habeebah157** [comment]
Hello, 
I am not sure if I am checking wrong but phone numbers would be under students right? Like the students numbers. I do it as one of the students fields. I also tried searching phone number in the codebase to give me a clue of what it could be but could not find it. There is nothing like phone number in the oyster channel on slack either.

**Comment by tomas-salgado** [comment]
Just left a comment about this in your Slack message, but basically PR is partly dependent on issue #477, since that issue will add the field `phone_number` to the database. While that is still in progress, you can add the field locally by adding the migration script [here](https://github.com/colorstackorg/oyster/blob/69158efab94dc84d42d2ff23182e134ac6742c2f/packages/db/src/migrations/20240824172428_add_phone_number_field.ts) (which is from the PR that's up for issue #477).

**Comment by wflore19** [comment]
Are there any updates on this issue @Habeebah157 ? I'd like to help

**Comment by Boris713** [comment]
^^

**Comment by tomas-salgado** [comment]
Assigning this to @wflore19!

**Comment by wflore19** [comment]
@Boris713 wanna pair program on this issue?

**Comment by Boris713** [comment]
@wflore19 I'm down if you are

**Comment by wflore19** [comment]
@Boris713 Habeebah finished this issue and is waiting to submit a pull request

**Comment by Habeebah157** [comment]
What is it now, are you guys still doing it? Is it still assigned to me? I can still make a pull request right? Sorry I am just seeing this.

**Comment by Habeebah157** [comment]
Usually how much time are we allowed to spend on an issue before it is transferred to someone else? I was not aware there was a time constraint or anything

**Comment by tomas-salgado** [comment]
The ideal timeframe is technically about 2 weeks, though it has been lenient

**Comment by tomas-salgado** [comment]
The ideal timeframe is about 3-4 weeks

**Comment by Habeebah157** [comment]
Oh, ok. Thank you. I was not aware of that. Sorry it took so long. I will try to make a pull request asap

**Comment by Habeebah157** [comment]
Hello, I had a problem with making a pull request and @wflore19  was able to help me solve it. I still have the code and all but I just want to let you guys know that it could take about a week to make another pull request because of my finals this week.
